### PR TITLE
feat(RichTextEditor): add checkboxes

### PR DESCRIPTION
**Pull request by Walkeezy**
This PR implements the checkbox functionality from this ticket: https://app.clickup.com/t/22wjrj2

### CODE REVIEW

File: src/components/RichTextEditor/RichTextEditor.stories.tsx [hunk 1]
```diff
@@ -135,6 +136,24 @@ const value = [
             },
         ],
     },
+    {
```
Comment (NadiaPosch) [inline src/components/RichTextEditor/RichTextEditor.stories.tsx:139]: I'd move the example values that are used in the stories into a separate file to keep the story file short and organised, wdyt?

File: src/components/RichTextEditor/components/checkbox-item.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                    checked={!!checked}
+                    onChange={(e) => {
+                        const path = ReactEditor.findPath(editor, element);
+
+                        setNodes<TElement<TodoListItemNodeData>>(
+                            editor,
+                            { checked: e.target.checked },
+                            {
+                                at: path,
+                            },
```
Comment (NadiaPosch) [inline src/components/RichTextEditor/components/checkbox-item.tsx:28]: I find the formatting a bit weird here, maybe it changes if you remove the comma after `path`?

Comment (Walkeezy) [reply src/components/RichTextEditor/components/checkbox-item.tsx:28]: Changed

File: src/components/RichTextEditor/RichTextEditor.stories.tsx (hunk 1 above)
Comment (Walkeezy) [reply src/components/RichTextEditor/RichTextEditor.stories.tsx:139]: Good input, refactored it :)

File: src/components/RichTextEditor/utils/alignment.ts [hunk 3]
```diff
@@ -1,3 +1,5 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```
Comment (NadiaPosch) [inline src/components/RichTextEditor/utils/alignment.ts:1]: 😄

**Review by NadiaPosch — APPROVED** [review]

**Pull request merged**